src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -54,6 +56,7 @@ public Shooter() {
         new CANSparkMax(ShooterConstants.kBottomFlywheelMotorId, MotorType.kBrushless);
     m_bottomFlywheelMotor.follow(m_topFlywheelMotor, true);
     m_bottomFlywheelEncoder = m_bottomFlywheelMotor.getEncoder();
+    m_lineBreakSensor = new DigitalInput(ShooterConstants.kLineBreakPort);
```

Comment:
Any particular reason the line break sensor was moved to the Shooter subsystem? I feel like it works best in the Indexer or possibly Intake subsystem because that is where the sensor is primarily used.